### DISCUSSION TITLE
authentication through RSA in SPDM

### DISCUSSION THREAD

**Discussion by PoojaGaur1708** [DMTF/libspdm#846] [Q&A] [answered]
Hello,
 
For RSA authentication , what msg to be sent , field will need to be set between requester and responder.

**Comment by steven-bellock** [comment, marked as answer]
First, the Requester should send only RSA algorithms during `NEGOTIATE_ALGORITHMS`; hopefully the Responder supports them. Then authentication is achieved via the `CHALLENGE` / `CHALLENGE_AUTH` request / response.
